pysiaf/tests/test_nirspec.py

```diff
@@ -33,7 +33,12 @@ def test_against_test_data(siaf=None):
 
     """
     if siaf is None:
-        siaf = Siaf(instrument)
+        # Try to use pre-delivery-data since this should best match the source-data. If no data there, use PRD data
+        try:
```

Comment:
I would have used a direct check using `os.path.isfile` but this implementation is fine too.